src/__tests__/log.test.ts

```diff
@@ -0,0 +1,48 @@
+import { ILogMsg, Logger } from '../common/logger';
+
+describe('logger', () => {
```

Comment:
@vrbsm It would be good to include test cases to verify log data. Example: 
### Log
`logger.info('verifying is first launch', { appVersionString, userConfigVersion });`

### Result
`[info] verifying is first launch, 'object': {"appVersionString":"4.3.0","userConfigVersion":"4.3.0"}`

### Validation
`expect(logger.log).toHaveBeenCalled('info', 'verifying is first launch', [{"appVersionString":"4.3.0","userConfigVersion":"4.3.0"}]);`